lib/rubocop/rspec/config_formatter.rb

```diff
@@ -27,16 +27,21 @@ def unified_config
         cops.each_with_object(config.dup) do |cop, unified|
           next if SUBDEPARTMENTS.include?(cop)
 
-          unified[cop] = config.fetch(cop)
```

Comment:
Is there a point to `fetch` from `config` when `unified` is `config.dup`?

Reply:
Nope. Good catch.